### PR TITLE
Feature/cron script

### PR DESCRIPTION
**Pull request by chemaper**
#### Reviewers
 @flopezag
 
 #### Description
Changes, including documentation, to allow executing all the steps using a cron script.

Every day:
*users next to expire are notified
*users expired are changed to basic and their VMs stopped
*resources of users changed to basic N days ago (a rotation is used) are freed.

 #### Testing
 Locally

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -47,6 +47,47 @@ by impersonating the users themselves. This is the only way to delete the key pa
...
 
+The scripts can be invoked in two ways:
+
+* automatically, as a cron daily script
+* manually
+
+The automatic way: cron script
+------------------------------
+
+This is the recommended use of the script. Every day, the script notifies the
```
Comment (flopezag) [inline README.rst]: ...the script notifies the next users to expire...

File: README.rst [hunk 2]
```diff
@@ -100,6 +141,36 @@ installation.
...
+  export OS_USERNAME=<admin_user>
+  export OS_TENANT_NAME=<admin_tenant>
+  export OS_PASSWORD=<passwrod_admin>
+  export OS_AUTH_URL=<keytone_url>
+
+  export TRUSTEE_USER=<trustee_user>
+  export TRUSTEE_PASSWORD=<trustee_password>
+  /root/fiware-skulds/cron-script.sh
+
+It is recommended to do this file only readable by the root user, because it
```
Comment (flopezag) [inline README.rst]: It is recommended to make...

File: cron_script.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+cd $(dirname $0)
+
+. deleteENV/bin/activate
```
Comment (flopezag) [inline cron_script.sh]: it is correct a . follow by blank space and execution of a script?

File: cron_script.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+# stop servers of expired users
+if [ ! -f users_to_delete_phase3.txt ] ; then
+./phase1_generate_trust_ids.py
+./phase2_stopvms.py
+rm users_trusted_ids.txt
+fi
+
+# delete resources of users who expired days ago (i.e. accounts that were
+# processed by the phase2_stopvms some days ago)
+if [ -f users_to_delete_phase3.txt ] ; then
```
Comment (flopezag) [inline cron_script.sh]: this is not the else operation of the previous condition?

But if you want to check it again due to it is possible that there is Nothing to delete, then you should give a (log) message in order to inform that there is no users to delete in phase3 or there was an uncontrolled error.

File: cron_script.sh (hunk 3 above)
Comment (chemaper) [inline cron_script.sh]: Yes, it is. This is for loading the environment variables in the caller environment. If you invoke with deleteENV/bin/activate, the environment variables are defined in the called script.

This line is required for using the virtualenv.

File: cron_script.sh (hunk 4 above)
Comment (chemaper) [inline cron_script.sh]: Uhm, I see the script is wrong. The conditions must be:

 if [ -f users_to_delete.txt ] ; then
  # stop servers
fi

if [ -f users_to_delete_phase3.txt ] ; then
 # delete resources
fi

File: cron_script.sh (hunk 3 above)
Comment (flopezag) [inline cron_script.sh]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**